x-pack/test/cases_api_integration/common/lib/api/case.ts

```diff
@@ -7,12 +7,14 @@
 
```

Comment:
do you think it would be beneficial to add check if this rule action is displayed on rule actions UI?
https://github.com/elastic/kibana/blob/8.15/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_actions/rule_actions_pli_essentials.cy.ts

Reply:
added here: https://github.com/elastic/kibana/pull/183937/files#diff-b8a38d4495c5111f440c695ea309a25b30cac610e769c2938d0e7e25937c7486R72